Jolly.xcodeproj/GeneratedModuleMap/CHTTPParser/module.modulemap

```diff
@@ -0,0 +1,5 @@
+framework module CHTTPParser {
+    umbrella "/Users/pablovillar/Desktop/Projects/Jolly/Packages/CHTTPParser-0.3.0/Sources/CHTTPParser/include"
```

Comment:
After further investigation, it seems that we cannot avoid this. 

These local paths in `modulemap` files are produced by [Swift Package Manager](https://swift.org/package-manager/), the new dependency manager that is native as of Swift 3 (we are leveraging this manager because of [Kitura](https://github.com/IBM-Swift/Kitura)).

The manager itself is still under constant review and not widely accepted yet. There isn't really much documentation and we expect these kind of issues to be solved in the future.

**Applied solution:**
- Ignore the `.xcodeproj` file. The xcode project can be manually generated, though, by running (locally):
```shell
$ swift package generate-xcodeproj
```

No further configuration is required to replicate the same xcode project environment in different machines.